gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/KubernetesApplyTaskTest.java

```diff
@@ -60,7 +60,7 @@ public void run_withImplementationPending_shouldThrowException() {
     // Given
     final KubernetesApplyTask applyTask = new KubernetesApplyTask(KubernetesExtension.class);
     // When
-    final UnsupportedOperationException result = assertThrows(UnsupportedOperationException.class, applyTask::runTask);
+    final UnsupportedOperationException result = assertThrows(UnsupportedOperationException.class, applyTask::run);
```

Comment:
Unless necessary, tasks should be tested using the runTask method. Not sure why you changed it here.

Reply:
I had updated AbstractJKubeTask::runTask method for BuildTask Dockerfile implementation. This test started failing due to it no longer throwing exception.

Reply:
Well, this is just as good as adding an `@Ignore`.